index.js

```diff
@@ -111,10 +111,14 @@ var addInstructionsToBrowserify = require('./lib/bundler');
  * of `{appRootDir}/middleware.json`
  * @property {Object} [components] Component configuration to use instead
  * of `{appRootDir}/component-config.json`
+ * @property {Array.<String>} [mixinSources] List of directories where to look
+ * for files containing model mixin definitions.
```

Comment:
If I remember our previous discussions correctly, the name `mixinSources` should be used for a list of directories which are scanned only for the mixins that are actually used by some model. As opposed to `mixinDirs`, that is pointing to directories where all files are loaded.

This distinction is very important in order to keep the size of the browser bundle as small as possible.

I am not asking you to implement the full "lazy" loading of mixins if you don't have bandwidth for that now, but I am asking you to rename the config option to keep it consistent with `bootDirs` and `modelSources`.